components/plurals/src/operands.rs

```diff
@@ -79,7 +79,8 @@ use fixed_decimal::FixedDecimal;
 ///     }),
 ///     FixedDecimal::from(12345)
 ///         .multiplied_pow10(-2)
-///         .map(|d| (&d).into())
```

Comment:
@sffc , I think it should be a better solution than converting to string and parse.